Packs/Tripwire/Integrations/Tripwire/Tripwire_description.md

```diff
@@ -0,0 +1,4 @@
+## Tripwire
```

Comment:
please add how to get the required creds or/and to set the necessary permissions.

Reply:
You just need your username and password

Reply:
ok. no special permissions needed?